src/components/DateTimePicker/Internal/Tests/util/commonUtil.tsx

```diff
@@ -177,29 +186,38 @@ export function selectCell(text: string | number, index = 0) {
 
 export function clickButton(type: string) {
   let matchBtn: HTMLButtonElement;
-  Array.from(document.querySelectorAll('button')).forEach((btn) => {
-    if (btn.className.includes(`-header-${type}-btn`)) {
-      matchBtn = btn;
-    }
-  });
 
-  fireEvent.click(matchBtn);
+  if (canUseDocElement()) {
+    Array.from(document.querySelectorAll('button')).forEach((btn) => {
+      if (btn.className.includes(`-header-${type}-btn`)) {
+        matchBtn = btn;
+      }
+    });
+
+    fireEvent.click(matchBtn);
+  }
 
   return matchBtn;
 }
 
 export function confirmOK() {
-  fireEvent.click(document.querySelector('.picker-ok > *'));
+  if (canUseDocElement()) {
```

Comment:
seems unlikely to need such a wrapper? can't imagine confirm being clicked on the server